types/eventsource/eventsource-tests.ts

```diff
@@ -0,0 +1,15 @@
+import * as EventSource from "eventsource";
```

Comment:
Use the `import EventSource = require("eventsource")` syntax for modules that use `export = EventSource`